src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -65,12 +65,14 @@ public void checkApiRateLimit(@NonNull TaskListener listener, GitHub github) thr
                                 - Math.max(0, (long) (targetFraction * MILLIS_PER_HOUR))
                                 + ENTROPY.nextInt(1000);
                         listener.getLogger().println(GitHubConsoleNote.create(System.currentTimeMillis(), String.format(
-                                "GitHub API Usage: Current quota has %d remaining (%d over budget). Next quota of %d in %s. Sleeping for %s.",
+                                "Jenkins-Imposed API Limiter: Current quota for Github API usage has %d remaining (%d over budget). Next quota of %d in %s. Sleeping for %s.",
                                 rateLimit.remaining, ideal - rateLimit.remaining, rateLimit.limit,
                                 Util.getTimeSpanString(rateLimitResetMillis),
                                 Util.getTimeSpanString(expiration - System.currentTimeMillis())
                         )));
                     }
+                    listener.getLogger().println(GitHubConsoleNote.create(System.currentTimeMillis(),
+                            "Jenkins is using \"Throttle for Normalize\" to try and evenly distribute GitHub API requests. Use \"Throttle on Over\" to have Jenkins restrict GitHub API requests only when near or above the rate limit."));
```

Comment:
Tests should include check for this message when it is expected.

Reply:
Does this message get printed even if no throttling occurs?

I would probably have this message and the equivalent for the other strategy mention where you need to go to change the setting (e.g. the GitHub section of the Global configuration page or wherever it is). Also, I don't think that "Throttle on Over" or "Throttle for Normalize" are ever shown to users anywhere, so I probably wouldn't use that terminology here, but I might be wrong. My suggested wording would be something like: "Jenkins is attempting to evenly distribute GitHub API requests. To configure a different rate limiting strategy, such as having Jenkins restrict GitHub API requests only when near or above the GitHub rate limit, go to ...".

Reply:
+1 on the messaging

This message will NOT get printed if no throttling occurs. Which is actually a different behavior from `ThrottleOnOver`. I should always have it printed. Will change.

Reply:
FWIW my preference would be to only want to print it if throttling _does_ occur to avoid spamming the logs in cases where there is no problem, under the assumption that if users aren't being throttled they have no reason to care about what strategy they are using.

Reply:
fair point, i think the reduction of logs outweighs that type of information